app/controllers/govspeak_preview_controller.rb

```diff
@@ -4,6 +4,8 @@ class GovspeakPreviewController < ApplicationController
   skip_before_action :verify_authenticity_token
 
   def to_html
-    render plain: GovspeakService.new.to_html(request.raw_post)
```

Comment:
I think it'd be better if we change `request.raw_post` into something Rails-y, that will probably solve the encoding issues as well. Can we change the JS so that Rails can access it as `params[:govspeak]`?

Reply:
Ah, I think this was a gap in my knowledge about Rails forms. Thanks for prompting me to look at it again - I'll have a go at fixing it.